Umami/ci.yml

```diff
@@ -0,0 +1,23 @@
+prepare:
+  steps:
+    - name: Install n
+      command: sudo npm i -g n
+    - name: Install latest nodejs version
+      command: sudo n stable
+    - name: Delete older nodejs version because it clashes in the Run CI command.
+      command: sudo n prune
+    - name: install latest NPM
+      command: sudo npm i -g npm
+    - name: Install Yarn
```

Comment:
yarn is already preinstalled in a codesphere workspace

Reply:
Removed This ✅